test/explain/algorithm/test_gnn_explainer.py

```diff
@@ -139,6 +147,62 @@ def test_gnn_explainer_classification(
     check_explanation(edge_mask_type, node_mask_type, explanation)
 
 
+@pytest.mark.parametrize('edge_mask_type', edge_mask_types)
+@pytest.mark.parametrize('node_mask_type', node_mask_types)
+@pytest.mark.parametrize('explanation_type', ['model', 'phenomenon'])
+@pytest.mark.parametrize('task_level', ['node', 'edge', 'graph'])
+@pytest.mark.parametrize('return_type', ['log_probs', 'probs', 'raw'])
+@pytest.mark.parametrize('index', [None, 2, torch.arange(3)])
+@pytest.mark.parametrize('multi_output', [False, True])
+def test_gnn_explainer_binary_classification(
+    edge_mask_type,
+    node_mask_type,
+    explanation_type,
+    task_level,
+    return_type,
+    index,
+    multi_output,
+):
+    model_config = ModelConfig(
+        mode='binary_classification',
+        task_level=task_level,
+        return_type=return_type,
+    )
+
+    explainer_config = ExplainerConfig(
+        explanation_type=explanation_type,
+        node_mask_type=node_mask_type,
+        edge_mask_type=edge_mask_type,
+    )
+
+    model = GCN(model_config, multi_output)
+
+    target = None
+    if explanation_type == 'phenomenon':
+        with torch.no_grad():
+            out = model(x, edge_index, batch, edge_label_index)
+            target = (out > 0.5).long()
```

Comment:
This needs to be dependent on the `return_value`.

Reply:
indeed!